Tasks/AzureMonitorAlerts/AzureRmRestClient.ts

```diff
@@ -0,0 +1,236 @@
+import * as tl from "vsts-task-lib/task";
+import * as Q from "q";
+import { 
+	HttpClient, 
+	HttpClientResponse 
+} from "typed-rest-client/HttpClient";
+
+import { 
+	RestClient, 
+	IRequestOptions, 
+	IRestResponse 
+} from "typed-rest-client/RestClient";
+
+import { AuthorizationClient } from "./AuthorizationClient";
+
+import * as Util from "./Utility";
+import { 
+	IAzureMetricAlertRequestBody,
+	IAzureMetricAlertRule
+} from "./Interfaces";
+
+export class AzureRmRestClient {
```

Comment:
This is not generic AzureRM class. This is very specific to Azure alerts. Can you rename accordingly.